### ISSUE TITLE
Use 7zip as the EXE wrapper instead of AutoIt

### ISSUE THREAD

**Issue by eesheesh** [google/gsps-support-tool#10]
We can use https://www.7-zip.org/a/lzma1900.7z from https://www.7-zip.org/sdk.html. We need to create `build.config.gspstool`:
```
;!@Install@!UTF-8!
ExecuteFile="cmd.exe"
ExecuteParameters="/c title G Suite Password Sync diagnostics tool && cscript //nologo gspstool.vbs"
Directory="%temp%"
GUIMode="2"
Progress="no"
;!@InstallEnd@!
```

...and then:
```
"C:\Program Files\7-Zip\7z.exe" a gspstool.7z gspstool.vbs
copy /b 7zSD.sfx  + build.config.gspstool + gspstool.7z test-exe\gspstool.exe
```

**Comment by eesheesh** [comment]
Or just use [AutoHotKey](https://www.autohotkey.com/docs/misc/Ahk2ExeDirectives.htm).

**Comment by eesheesh** [comment]
Fixed in 885078a, using the pure VBS method.
